redis/src/sentinel.rs

```diff
@@ -141,6 +149,10 @@ pub struct SentinelNodeConnectionInfo {
 
     /// The Redis specific/connection independent information to be used.
     pub redis_connection_info: Option<RedisConnectionInfo>,
+
+    /// Certificates to be used, or none if we want to use certificates from local truststore
+    #[cfg(feature = "tls-rustls")]
+    pub certs: Option<TlsCertificates>,
 }
 
 impl SentinelNodeConnectionInfo {
```

Comment:
please add setters for all of `SentinelNodeConnectionInfo`'s fields.
We should've done it beforehand, sorry :(

Reply:
Do we actually need setters? I'm not clear what it benefits here.

Reply:
adding a public field to a public struct is a breaking change, since it breaks all explicit initializations. Setters should be used instead of public fields, so that the `Foo::new().set_bar(bar)` structure is never broken.

Reply:
I'm not a huge fan of patterns that require mutability. Builders are the right choice here (and for all the clients).

Reply:
ah, sorry. yes, I meant the builder pattern. my bad :)